### ISSUE TITLE
Feedback on Gravitational Wave Sources Section 6.5

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTScienceCollaborations/ObservingStrategy#435]
Hi @soares-santos  Great to see you the other day, thanks for agreeing to help with the white paper internal review :-) It would be great if you could take a look at the Gravitational Wave Sources Section 6.5 of the Transients chapter that you started, and give the writing team (including @raffaellamargutti  and @zodoctor) some feedback, via this issue or other new ones. Our internal review milestone has a due date of June 3rd, so you'll have until the end of next week - although the sooner you can get back to us, the sooner we can get this section finished and ready to submit :-)

What we are looking for at this stage is to address the following questions about each science section:
1. Does the chapter tell a coherent story about the science that is being planned? 
2. Each science section (such as Section 6.5) needs to briefly describe a planned science project, including the primary measurement or discovery that is being sought. How well we expect to achieve this science goal needs to be quantified by a Figure of Merit, which will depend on the details of the observing strategy. Is it clear what the Figure of Merit is, for each science case? There are other, "diagnostic" metrics that help us understand how well the observing strategy will permit the science to be done. Are these defined and explained clearly?  
3. We are working towards calculating Figures of Merit given various simulated observing strategies, captured in the output databases of the `OpSim` operations simulator. Some science sections will contain  metric analyses of these outputs (which are sometimes referred to as "cadences"). What comments do you have on these analyses? In the cases where the Figures of Merit and/or diagnostic metrics have been defined, but not yet calculated, is a plan for future work presented? 
4. We expect each science section to close with a brief discussion of how the observing strategy could be changed to improve the forecast results, and of how this might impact other science cases. Is this provided, and does it make sense?

If you find any of these pieces missing, it'd be great to write an issue asking for them;  likewise, if anything needs cleaning up, explaining better, or otherwise fixing, please do issue those things too. You can also give high level feedback on this issue thread. You can get the [v0.8 internal review edition PDF file here](https://github.com/LSSTScienceCollaborations/ObservingStrategy/raw/issue/415/draft-for-review/whitepaper/LSST_Observing_Strategy_White_Paper.pdf), or just comment on the current version in the `master` branch (no-one is editing this section right now). 

Thanks very much for helping out with this! :-)

**Comment by soares-santos** [comment]
Hi @raffaellamargutti, @zodoctor, et al.:
 
I am glad to see that this effort is going forward. 
I will group my comments according to the 4 points that @drphilmarshall highlighted:

1. Does the chapter tell a coherent story about the science that is being planned?

The chapter in general does a good job of presenting the science case at hand, and makes the case for the need of a TOO program (without presenting a concrete proposal for how that would happen, but at this stage, this is OK). We can improve significantly the argument, though: 

What is missing here is a review of what exists in terms of current programs that search for optical counterparts of GW events. That can help inform the thinking about the problem: e.g. with DECam we developed a search that is sensitive to KN out to 30—200Mpc (depending on details of the models assumed; these numbers are for a range of peak brightness varying from i-band absolute magnitude M_i = -11 to -15; see Soares-Santos et al. 2016). 

What we learned from that exercise is that i,z bands are great discriminant against other transient contaminants, and that, if you have pre-existing templates, we can perform an efficient and clean search using 3-4 observation epochs, starting as early as possible and with separation of ~2 nights between them, plus one epoch 3+ weeks after (to reject SNe). 

LSST will have templates everywhere soon after it starts. Also, LSST will automatically take care of the exposure 3+ weeks after the event. So we could estimate what would be the extent of the TOO effort as: 2 filters x 4 epochs x 100 deg^2 / (9 deg^2 per pointing) = 67 exposures (x2 if we want more than one tiling to mitigate the effect of gaps). This could be consider an upper limit on the observing times needed. 
We can say that after the first epoch (or maybe the first 2?) we can already identify a preliminary list of candidates and then farm them out to a spectroscopic followup program. We can also probably identify areas where there are no candidates and drop them from the following nights of observations. This would mitigate the impact of this particular TOO program in the overall LSST strategy. 

2. Figure of Merit

The OpSim Analysis in 6.5.2 is good, but doesn't really go all the way to write a metric and evaluate it. 
For now I think this is OK, as it is just the starting point of this study. 

3. Figure of Merit plans

We didn't outline a FoM calculation plan. One plan that we could present here would be to do the exercise of evaluating the efficiency of detection, using similar criteria/methodology that we used in Soares-Santos et. al 2016 and in Doctor et al. 2016 (in prep.). We could evaluate the schema that I have outlined above and test the effect of using only the information obtained in the first epoch to see if we can get a usable list of preliminary candidates. Our FoM would be simply the probability of detection as a function of distance to the event, evaluated for a range of KN models. I don't think we can do this work in time for the review of this paper, but we can propose to do it for the near future. I'd be happy to help with that

4. Discussion of improvements

I think the main point, which is we need a TOO program to realize the full potential of GW data, is made very clear. We can put some of the details that I discussed above, to make the discussion more concrete.

**Comment by bethwillman** [comment]
I'm glad to see Marcelle's comments here.  Thank you!  Also tagging @tonytyson to be sure he has a chance to take as look at Section 6.5 of the White Paper as well, if he hasn't already.  Tony has been thinking a lot about possible LSST-GW synergies, and some of the challenges.

**Comment by jhrlsst** [comment]
Sorry. It was just an accident (pushing a wrong button).

**Comment by tonytyson** [comment]
A problem with this section is that it underestimates the required LSST observing because of the false alarm rate from all the other optical transients encountered during the GW EM counterpart ToO campaign.  This issue was addressed by Nissanke and Kasliwal (2013) and in a recent study I did (Doc-21432) available at
https://dl.dropboxusercontent.com/u/86228436/LSST%20GW%20Followup.pdf

**Comment by ebellm** [comment]
@tonytyson If I can interject my perspective as chapter editor: I see the major goal of this section as justifying the need for TOO observations of EM/GW triggers, over and above the universal cadence, rather than determining the appropriate followup strategy for LSST.  The latter will surely require detailed study and refinement in the coming years as the LIGO era advances.  However, what OpSIM runs can already tell us is what fraction of disruptive TOOs can be tolerated without compromising LSST's other science goals.

I also think your memo is too pessimistic about LSST ability to identify a potential counterpart (assumming nature produces kilonova with the theorized properties!).  By 2022, it is highly probable that there will be five interferometers operating (LIGO Livingston and Hanford, Virgo, KAGRA, and LIGO India), rather than the three your memo assumes.  With five interferometers the majority of localizations are of order 10 square degrees, not 100 (e.g., Nissanke, Kasliwal, and Georgieva 2013; Fairhurst 2012).  That sharply reduces both the cost of TOO observations as well as the number of unrelated transients.

**Comment by tonytyson** [comment]
Hi Eric,

It is unlikely in the extreme that by 2022 there will be 5 GW detectors 
with a-LIGO or better sensitivity observing simultaneously. KAGRA is way 
behind, and have encountered problems.  India has not committed yet, and 
the technical manpower is not there. So we need to stick with what we 
reasonably believe [LIGO + Virgo]: 100 sq.deg, on average.

My calculation addresses not the long term followup but rather the 
immediate LSST ToO response strategy which would be required to find a 
small shortlist of a few EM candidates which then could be immediately 
followed up by rapid multi-color optical monitoring and spectroscopy.

-Tony

**Comment by soares-santos** [comment]
@tonytyson @ebellm I am attaching a plot made by Hsin-Yu Chen, a collaborator from U Chicago. The plot shows the median localization area as a function of enclosed confidence level for the case of two (LIGO-H,LIGO-L) and three detectors (LIGO-H,LIGO-L,VIRGO). The median area for 90% c.l. is expected to be around 60 sq-deg for the case of 3 detectors. About 40% less than quoted above.  
<img width="763" alt="chen-2015-optimization-notes" src="https://cloud.githubusercontent.com/assets/4348386/15907592/6bcb2876-2d83-11e6-8b28-cfd7500cd94a.png">

The combination of color and time scale information for the transients is very efficient in reducing the backgrounds. We have done a study in DES SN fields (@zodoctor is currently preparing a draft of a paper that we plan to publish soon) demonstrating that we can develop a reasonably clean search. And our followup of GW150914 also shows that. Of course we have not yet done a specific study for LSST, but extending that work done in Doctor et al. to a detailed study of various proposed observing strategies would be a natural next step. I'd be happy to pursue that, if you are interested.

**Comment by ebellm** [comment]
Hi Tony,

Since LSST will only follow up a subset of LIGO triggers, we can restrict ourselves to the best-localized.  Even in the three-interferometer case, the best-localized 10% are closer to 10 square degrees than 100 (e.g., Nissanke+ 2013, Figure 5a).

Agreed that we can't be sure when KAGRA and LIGO-India will come online--but even if it's not until 2030, LSST will still have a role to play.

Best regards,
Eric

**Comment by tonytyson** [comment]
Hi,

This will depend strongly on GW S/N.   What was the assumed S/N?  It 
also depends on the GW filter. Does this have a optimal filter for some 
assumed source waveform, or is it a more open filter?

Thanks,
Tony

**Comment by raffaellamargutti** [comment]
Hi all
thanks for all the comments to the little piece that we wrote for the white paper. The goal of the gravitational wave section is just to show that if we want LSST to play a role in the world of EM counterparts to GW, then a ToO capability is needed. I was at a workshop at Columbia 10 days ago  and I was happy to hear discussions led by Tony about the actual strategy to use to maximize the scientific impact of LSST to GW-EM science.  I think this is indeed the next topic our subgroup should focus, now that it seems that ToOs will happen!
Raf
